config/samples/distributed-compute_v1alpha1_sparkcluster.yaml

```diff
@@ -59,7 +59,7 @@ spec:
 
   # KeyTab Config for Kerberos
   #  kerberosKeytab:
-  #    path: /tmp/keytab.conf
+  #    mountPath: /tmp/keytab.conf
   #    configs: dGVzdHZhbHVlCg==
```

Comment:
`configs` should be renamed to `contents`?

Reply:
yep, my mistake!